ps_accounts.php

```diff
@@ -363,13 +363,25 @@ public function getContent()
         /** @var \PrestaShop\Module\PsAccounts\Presenter\PsAccountsPresenter $psAccountsPresenter */
         $psAccountsPresenter = $this->getService(\PrestaShop\Module\PsAccounts\Presenter\PsAccountsPresenter::class);
 
+        /** @var \PrestaShop\Module\PsAccounts\Service\PsAccountsService $psAccountsService */
+        $psAccountsService = $this->getService(\PrestaShop\Module\PsAccounts\Service\PsAccountsService::class);
+
         Media::addJsDef([
             'contextPsAccounts' => $psAccountsPresenter->present((string) $this->name),
+            'contextInitParams' => $psAccountsService->getInitParams((string) $this->name),
```

Comment:
It will be accessible on the `window` object so the name should mention psAccounts, I tought first it would be included in the legacy context ?

Reply:
After looking twice it's already included in the PsAccountsPresenter